scss/detail.scss

```diff
@@ -0,0 +1,31 @@
+@import 'equalizer';
+@import 'styles';
+
+.book-detail {
+
+    .book-title {
+        margin: 0 0 52px 0;
```

Comment:
The indentation in all this file should be of 2 spaces each. Check this [style guide](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#indentation)